### ISSUE TITLE
ComboBoxAccessibleObject_ExpandCollapse_Set_CollapsedState failing

### ISSUE THREAD

**Issue by hughbe** [dotnet/winforms#3406]
* .NET Core Version: 
Master

* Have you experienced this same bug with .NET Framework?: 
N/A

**Problem description:**
```
Assert.Equal() Failure\r\nExpected: Expanded\r\nActual: None
   at System.Windows.Forms.Tests.AccessibleObjects.ComboBoxAccessibleObjectTests.ComboBoxAccessibleObject_ExpandCollapse_Set_CollapsedState(ComboBoxStyle comboBoxStyle) in F:\workspace\_work\1\s\src\System.Windows.Forms\tests\UnitTests\AccessibleObjects\ComboBoxAccessibleObjectTests.cs:line 35
```
https://dev.azure.com/dnceng/public/_build/results?buildId=674902&view=ms.vss-test-web.build-test-results-tab&runId=20913492&resultId=101913&paneView=debug

**Comment by RussKie** [comment]
Another one here - https://dev.azure.com/dnceng/public/_build/results?buildId=676108&view=results

**Comment by weltkante** [comment]
@RussKie I'm confused why the CI build you linked is considered successful even though a test failed (also in the "run unit test" logs there is no output of failed tests either). Did you manually trigger a rerun and it overwrote part of the CI output?

**Comment by RussKie** [comment]
I did re-run it, and hence it is green now.

![image](https://user-images.githubusercontent.com/4403806/84227514-f9b9d080-ab27-11ea-839b-2a2942040993.png)

https://dev.azure.com/dnceng/public/_build/results?buildId=676108&view=logs&j=1f132584-ab08-5fbf-e5d0-5685de454342&s=ff05ad62-bb9a-53b6-ce9f-72f329a63e7c
But with a re-run all logs are overwritten sadly.

I've checked the failed build didn't have any memdumps associated with it.

**Comment by weltkante** [comment]
> I've checked the failed build didn't have any memdumps associated with it.

You only get them for actual crashes, not for ordinary test failures.

**Comment by RussKie** [comment]
Closing as we no longer have access to the build results.
Will reopen if the error reappears.
